Composer/packages/lib/code-editor/src/BaseEditor.tsx

```diff
@@ -188,7 +188,7 @@ const BaseEditor: React.FC<BaseEditorProps> = (props) => {
         onBlurListener.dispose();
       };
     }
-  }, [editorRef.current]);
+  }, [editorRef.current, onBlur, onFocus]);
```

Comment:
One thing that could catch this is the eslint rule for exhaustive deps. We should look into enabling that.